crates/wasi-component-loader/src/extension/api/since_0_10_0/instance/hooks.rs

```diff
@@ -1,16 +1,21 @@
 use futures::future::BoxFuture;
 
-use crate::extension::HooksInstance;
+use crate::{SharedContext, extension::HooksInstance};
 
 impl HooksInstance for super::ExtensionInstanceSince0_10_0 {
     fn on_request(
```

Comment:
for resolver I just ended up having default implementation of the methods, just too painful to update everytime